html/Mogo/js/sections-animation/helper-functions.js

```diff
@@ -1,62 +1,53 @@
+let boxToHightlight;
+export function setNavbarBackground(navbar) {
+  navbar.classList.add('navbar--with-bg');
+}
 
-components.then((response) => {
-  const {navbar, navbarMenu} = response;
+function removeNavbarBackground(navbar) {
+  navbar.classList.remove('navbar--with-bg');
+}
 
-  document.addEventListener('DOMContentLoaded', () => {
-    function setNavbarBackground() {
-      navbar.classList.add('navbar--with-bg');
-    }
+export function removeMenuItemBackground() {
+  if (boxToHightlight) {
+    boxToHightlight.classList.remove('navbar__menu-item--active');
+  }
+}
 
-    function removeNavbarBackground() {
-      navbar.classList.remove('navbar--with-bg');
-    }
+export function changeNavbarBackground(navbar, navbarMenu) {
+  if (window.scrollY >= 200) {
+    setNavbarBackground(navbar);
+  } else if (!navbarMenu.classList.contains('navbar__menu--active')) {
+    removeNavbarBackground(navbar);
+  }
+}
 
-    function changeNavbarBackground() {
-      if (window.scrollY >= 200) {
-        setNavbarBackground();
-      } else if (!navbarMenu.classList.contains('navbar__menu--active')) {
-        removeNavbarBackground();
-      }
-    }
+export function scrollToSection(eventTarget) {
+  if (eventTarget.getAttribute('href') === '#') {
+    return;
+  }
+  const elementName = eventTarget.getAttribute('href');
+  const element = document.querySelector('#' + elementName);
+  window.scrollTo({
+    behavior: 'smooth',
+    left: 0,
+    top: element.offsetTop,
+  });
+}
 
-    function addScrollEffect(event) {
-      if (event.getAttribute('href') === '#') {
-        return;
-      }
-      const elementName = event.getAttribute('href');
-      const element = document.querySelector('#' + elementName);
-      window.scrollTo({
-        behavior: 'smooth',
-        left: 0,
-        top: element.offsetTop,
-      });
+export function addEventToMenuItems(navbarMenu) {
+  navbarMenu.addEventListener('click', (event) => {
+    event.preventDefault();
+    if (boxToHightlight) {
+      boxToHightlight.classList.remove('navbar__menu-item--active');
     }
-
-    function addEventToMenuItems() {
-      let boxToHightlight;
-      navbarMenu.addEventListener('click', (event) => {
-        event.preventDefault();
-        if (boxToHightlight) {
-          boxToHightlight.style.background = 'none';
-        }
-        let target = event.target;
-        if (target.tagName === 'A') {
-          addScrollEffect(event.target);
-          boxToHightlight = target.parentElement;
-        } else if (target.tagName === 'LI') {
-          addScrollEffect(event.target.firstElementChild);
-          boxToHightlight = target;
-        }
-        boxToHightlight.style.background = 'rgba(121, 121, 121, 0.8)';
-      });
+    let target = event.target;
+    if (target.tagName === 'A') {
+      scrollToSection(event.target);
+      boxToHightlight = target.parentElement;
+    } else if (target.tagName === 'LI') {
```

Comment:
you could change the way you done it and rid of this if (a elements is a block with paddings and so on, li block is just container)